ocs_ci/framework/conf/ocp_version/ocp-4.9-config.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# Config file for nightly OCP 4.9
+RUN:
+  client_version: "4.9.0-0.nightly"
+DEPLOYMENT:
+  installer_version: "4.9.0-0.nightly"
+  terraform_version: "0.12.26"
+  # ignition_version can be found here
+  # https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks
+  ignition_version: "3.2.0"
+ENV_DATA:
+  vm_template: "rhcos-48.83.202103221318-0-vmware.x86_64"
```

Comment:
Maybe worth create issue, link it here in comment that once there will be RHCOS 4.9 that we should replace it.
And I am thinking about to move 4.9 things out of this PR and make separate PR for that. WDYT @vavuthu

Reply:
Done